embulk-core/src/main/bat/selfrun.bat

```diff
@@ -84,11 +84,21 @@ if "%optimize%" == "true" (
     set java_args=-XX:+AggressiveOpts -XX:+TieredCompilation -XX:TieredStopAtLevel=1 -Xverify:none %java_args%
 )
 
-java %java_args% -jar %this% %jruby_args% %args%
-
-endlocal

```

Comment:
This `endlocal` is removed, and the first `setlocal` does not look to be closed. Is the `setlocal`-`endlocal` mapping correct?